src/components/UnlayerEditorWrapper.tsx

```diff
@@ -145,6 +145,7 @@ export const UnlayerEditorWrapper = ({
       sort: false,
     },
     displayMode: "email",
+    version: "l.5.75",
```

Comment:
Is the `l` intentional or should it be a `1`?

```suggestion
    version: "1.5.75",
```

Reply:
is 1. thanks, is the l of lastest 😔